scss/product/components/_banner.scss

```diff
@@ -50,6 +56,8 @@
 @each $color,
 $value in $theme-colors {
     .banner-#{$color} {
+      @if $color != 'link'{
         @include banner-variant($value);
+      }
     }
```

Comment:
Are banner colors different from alert colors? If not, then you can use `$alert-colors` instead of `theme-colors` because alert colors are different than theme colors